ckanext/openbudgetsin_theme/templates/header.html

```diff
@@ -17,7 +17,7 @@
 
       {% block header_logo %}
         {% if g.site_logo %}
-          <a class="logo" href="{{ h.url('home') }}"><img src="{{ h.url_for_static_or_external(g.site_logo) }}" alt="{{ g.site_title }}" title="{{ g.site_title }}" /></a>
+          <a class="logo" href="{{ h.url('home') }}"><img src="{{ h.url_for_static_or_external(g.site_logo) }}" alt="Open Budgets India header logo" title="{{ g.site_title }}" /></a>
```

Comment:
The purpose of alt and title are different. Thus they aren't suppose to have same content. Also as alt is used highly in screen readers, SEO and for other purposes. Wanted to have a better description for the logo than just "OpenBudgetsIndia" which is the title.